dri3_ws.c

```diff
@@ -168,12 +173,14 @@ static struct driws_buffer *create_buffer(struct driws_drawable *drawable)
 
 	uint32_t stride, width, height;
 
+	int buffer_width = round_up_to(drawable->width, 4);
```

Comment:
uint32_t would be better for buffer_width and the round_up_to parameters.

Reply:
Fixed